scripts/installer/install

```diff
@@ -817,16 +817,26 @@ sub setup_glassfish {
 # add a check.
 
     print "\nInstalling the Glassfish PostgresQL driver... ";
-#    system( "/bin/cp", "pgdriver/" . $postgres_jdbc, $glassfish_dir . "/glassfish/lib" );
     my $pgdriver_success = copy("pgdriver/" . $postgres_jdbc, $glassfish_dir . "/glassfish/lib" );
 
-    unless ($pgdriver_success) 
+    unless ($pgdriver_success)
     {
 	print "\n*********************\n";
 	print "ERROR! Failed to copy the postgres driver into " . $glassfish_dir . "/glassfish/lib - check the directory permissions!\n";
 	exit 1;
     }
 
+	print "\nUpgrading eclipselink to v2.6.8 ... ";
+
+	my @fixedFiles = glob("glassfishfix/*.jar");
+
+    	for my $file (@fixedFiles) {
+    		copy($file, $glassfish_dir . "/glassfish/modules") or die "\nCopy failed for file: $file";
+    	}
+
+    	unlink $glassfish_dir . "/glassfish/modules/org.eclipse.persistence.moxy.jar" or die "\n moxy file couldn't be deleted";
```

Comment:
If deleting would happen only in case when file exists, then script could be executed many times without breaking anything

Reply:
That's a good point, but if we want fully fix it we would also have to take care of database. Right now if you re-run the script it will fail anyway since the database already exists.